### PR TITLE
Related #80 | Add video-to-text schema

### PR DESCRIPTION
**Pull request by akhdanfadh**
This PR adds a new schema for general video-to-text tasks, needed for MSVD-Indonesian dataloader in #80. This is specifically designed to support, but not limited to, video captioning and video-to-text retrieval.

Please note that the codes have been formatted using the SeaCrowd formatter (executed via `make check_file`) so there are changes in certain parts that are not directly related to the video schema implementation.

### CODE REVIEW

**Review by SamuelCahyawijaya — APPROVED** [review]
Hi @akhdanfadh, thank you for proposing and implementing the `video-to-text` schema. 

I think this is a good start for video dataset in general, especially since we couldn't find any standardized schema for such tasks.

**Review by holylovenia — CHANGES_REQUESTED** [review]
Hi @akhdanfadh, thanks for the new schema!! LGTM, but could you please pull the recent changes from the `master` branch? We'll merge your PR afterwards.

**Comment by sabilmakbar** [comment]
A quick qn:
Is it make sense to provide Video Link to it, unlike `Audio` dataset which HF provides it (and I believe its individual datapoint has been decoded)?

I don't think HF provides the defined `dataset.Feature` subclass for Video either
see here for the example of video data from HF (https://huggingface.co/docs/transformers/tasks/video_classification)

In any case, generally I'm okay w/ current implementation, just wondering whether this schema is quite sustain throughout the project for Video-modality NLP data...

**Comment by akhdanfadh** [comment]
@sabilmakbar 
Generally, for a video dataset, you need to download the entire videos locally for efficient processing. However, this does not limit the possibility of using only video URLs, such as in [this dataset](https://huggingface.co/datasets/HuggingFaceM4/webvid).

I'm a bit confused about what you mean by 'video link' here, as you compare it with Audio features. But yes, HF doesn't yet provide a feature for loading videos directly, and this is currently an open issue, as seen [here](https://github.com/huggingface/datasets/issues/5225).

As a workaround, I think dataloader implementers can use the 'video_path' feature to store a video URL, if applicable.

**Comment by akhdanfadh** [comment]
Hopefully done @holylovenia !

**Comment by sabilmakbar** [comment]
> @sabilmakbar Generally, for a video dataset, you need to download the entire videos locally for efficient processing. However, this does not limit the possibility of using only video URLs, such as in [this dataset](https://huggingface.co/datasets/HuggingFaceM4/webvid).

Okay, now that we have that as another example of Video Data loading, this schema shd be fine in design (at least until the PR of Videos has been implemented). However the caveat is, the URL itself must be publicly exposed (and video-downloadable) aside from its dataset containing Video Informations.

This needs extra attention and care on downloading fro URL given, as it might contain harmful and/or malicious file(s).

Btw, do you mind to add these caveats as a comment in this schema definition? (in `seacrowd/utils/schemas/video.py` L11)

> I'm a bit confused about what you mean by 'video link' here, as you compare it with Audio features. But yes, HF doesn't yet provide a feature for loading videos directly, and this is currently an open issue, as seen [here](https://github.com/huggingface/datasets/issues/5225).

Video Link ~ Video URL.

File: seacrowd/utils/schemas/video.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+General Video-to-Text Schema, including:
+- Video captioning
+- Video to text retrieval
+"""
+import datasets
+
+features = datasets.Features(
+    {
+        "id": datasets.Value("string"),
+        "video_path": datasets.Value("string"),
```
Comment (sabilmakbar) [inline seacrowd/utils/schemas/video.py:11]: add some notes here according to the discussion regarding URL must be publicly accessible, downloadable to obtain its videos, and safe from malicious files

**Comment by akhdanfadh** [comment]
Done @sabilmakbar

**Review by sabilmakbar — APPROVED** [review]
LGTM, thanks!

**Comment by sabilmakbar** [comment]
Hi @holylovenia , if you had the time, would you mind quickly checking this one? I believe he has done what you asked for

**Review by holylovenia — APPROVED** [review]
Thanks @akhdanfadh!! Looks perfect. Merging it now. Thanks @sabilmakbar for letting me know too.

**Pull request merged**